.env.template

```diff
@@ -31,16 +31,16 @@ POSTGRESQL_HOST=localhost
 POSTGRESQL_PORT=5432
 POSTGRESQL_USER_SUFFIX=
 PGSSLMODE=disable
-POSTGRESQL_ROOT=postgres
-POSTGRESQL_ROOT_PASSWORD=postgres
+POSTGRESQL_ROOT=root
```

Comment:
As already mentioned in another comment below, please revert the credentials values, Such change is only needed for PR builds and would require developers to re-create their local databases without an explicit need. This should be reflected in PR/Build-specific files.